### PR TITLE
Action: Add android build set-up composite action

### PR DESCRIPTION
**Pull request by niley7464**
This patch adds composite action for set up android build.

### CODE REVIEW

File: .github/actions/set-up/action.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        java-version: 17
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v3
+    - run: |
+        sudo apt-get update --fix-missing
+        sudo apt-get remove -y '^dotnet-.*'
+        sudo apt-get remove -y '^llvm-.*'
+        sudo apt-get remove -y '^mysql-server-core-.*'
+        sudo apt-get remove -y '^postgresql-.*'
+        sudo apt-get remove -y azure-cli google-chrome-stable google-cloud-cli firefox powershell microsoft-edge-stable mono-devel
```
Comment (wooksong) [inline .github/actions/set-up/action.yml:25]: https://www.ecloudcontrol.com/best-practices-to-reduce-docker-images-size/

Comment (niley7464) [reply .github/actions/set-up/action.yml]: Updated!
Thank you :)

File: .github/actions/set-up/action.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        sudo rm -rf /usr/share/dotnet/
+        sudo rm -rf /usr/share/swift
+        sudo rm -rf /usr/share/miniconda
+        sudo rm -rf /usr/local/graalvm/
+        sudo rm -rf /usr/local/.ghcup/
+        sudo rm -rf /usr/local/share/powershell
+        sudo rm -rf /usr/local/share/chromium
+        sudo rm -rf /usr/local/lib/node_modules
+        sudo rm -rf /var/lib/docker
+        sudo rm -rf /var/lib/apt/lists
```
Comment (wooksong) [inline .github/actions/set-up/action.yml:35]: Every (almost) ```RUN```  or ```CMD```line that affects the file system in a docker file increases the number of layers. Therefore, I think using ```&&``` and squashing the lines is better.

Comment (niley7464) [reply .github/actions/set-up/action.yml]: I missed this part 😅
Thank you :)

**Review by wooksong — APPROVED** [review]
LGTM!

**Pull request merged**